src/site/content/en/progressive-web-apps/what-are-pwas/index.md

```diff
@@ -0,0 +1,135 @@
+capable, reliable, and installable. These three pillars transform them into an
+experience that feels like a native application.
+
+### Capable
+
+The web is quite capable in its own right today. You can build a hyper-local
+video chat app using WebRTC, geolocation, and push notifications. You can make
+that app installable and take those conversations virtual with WebGL and WebVR.
+With the introduction of Web Assembly, developers can tap into other ecosystems,
+like C, C++, and Rust, and bring decades of work and capabilities to the web
+too! [Squoosh.app](https://squoosh.app/), for instance, leveraged this for its
+advanced image compression.
+
+Until recently, only native apps could really lay claim to these capabilities.
+While some capabilities are still out of the web's reach, new and upcoming APIs
+are looking to change that, expanding what the web can do with features like
+file system access, media controls, app badging, and full clipboard support. All
+of these capabilities are built with the web's secure, user-centric permission
+model, ensuring that going to a website is never a scary proposition for users.
+
+Between modern APIs, Web Assembly, and new and upcoming APIs, web applications
+are more capable than ever, and those capabilities are only growing.
+
+### Reliable
+
+A reliable Progressive Web App feels fast and dependable regardless of the
+network.
+
+Speed is critical for getting users to _use_ your experience. In fact, as page
+load times go from 1 second to ten seconds, the probability of a user bouncing
+[increases by 123%](https://www.thinkwithgoogle.com/marketing-resources/data-measurement/mobile-page-speed-new-industry-benchmarks/).
+Performance doesn't stop `onload`. Users should never wonder whether their
+interaction—for example, clicking a button—was registered or not. Scrolling and
+animation should feel smooth. Performance affects your entire experience, from
+how users perceive your application to how it actually performs.
+
+Finally, reliable applications need to be usable regardless of network
+connection. Users expect apps to start up on slow or flaky network connections
+or even when offline. They expect the most recent content they've interacted
+with, like media tracks or tickets and itineraries, to be available and usable
+even if getting a request to your server is hard. When a request isn't possible,
+they expect to be told there's trouble instead of silently failing or crashing.
+
+Users love apps that respond to interaction in the blink of an eye, and an
+experience they can depend on.
+
+### Installable
+
+Installed Progressive Web Apps run in a standalone window instead of a browser
+tab. They're launchable from on the user's home screen, dock, taskbar, or shelf.
+It's possible to search for them on a device and jump between them with the app
+switcher, making them feel like part of the device they're installed on.
+
+New capabilities get opened up once installed. Keyboard shortcuts usually
```

Comment:
'New capabilities get opened up once installed' --> 'New capabilities open up after a web app is installed.'

The flip side of 'omit needless words', is 'do not take shortcuts at the cost of clarity'. The original wording feels a bit like a shortcut.

Reply:
+1 this sentence sounded funky to me

Reply:
```suggestion
New capabilities open up after a web app is installed. Keyboard shortcuts usually
```